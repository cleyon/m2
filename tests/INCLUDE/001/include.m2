Outer 1
@include file1
Outer 2
