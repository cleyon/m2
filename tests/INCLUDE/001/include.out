Outer 1
FOO = 42
Outer 2
